readme and script update

ensonic opened the pull request:
Tried to convert a bunch of packs. A few small patches below to update the README and improve the script a bit.

ensonic committed:
README: update

add unzip to requirements and tell how to invoke the direct conversion.

ensonic committed:
mcresconvert: don't assume texture directory exists

Fixes #14

ensonic committed:
mcresconvert: add more error handling and early exits

Exit when unpacking the zip failed. Also exit if the 'texturepack' has no
textures. There are many resource packs available for download that have no
textures, even though they claim they have.

ensonic commented:
Sorry, I don't understand. I did not change anything w.r.t a default directory. The script is using '~/.minetest/textures' which is the default for the local mintest dir. It would not write the e.g. /usr{/local,}/share/minetest/ even if you run it as root.

sofar commented:
The script is written to fail when ~/.minetest does not exist. It is done so to prevent writing textures in folders that are irrelevant (in case people have installed minetest differently).

If you `mkdir -p ~/.minetest/textures`, then it is 100% guaranteed that the folder is *not* going to be found by any existing minetest installation on the system.

This is the reason why it currently is fatal. Making it non-fatal should at least print out a significant warning message.

ensonic commented:
Please take another look.

ensonic committed:
Improve code converting doors.

Extract code into function. Add workarounds for non-standart sizes. Handle
animated textures.

Fixes #16
